lib/geoserver/publish/layer.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+module Geoserver
+  module Publish
+    class Layer
+      attr_reader :connection
+
+      def initialize(conn = nil)
+        @connection = conn || Geoserver::Publish::Connection.new
+      end
+
```

Comment:
No `#create` because GeoServer layers cannot be created.

> To create a new layer, instead POST to one of /workspaces/{workspaceName}/coveragestores/{coveragestoreName}/coverages, /workspaces/{workspaceName}/datastores/{datastoreName}/featuretypes, /workspaces/{workspaceName}/wmsstores/{wmsstoreName}/wmslayers, or /workspaces/{workspaceName}/wmtsstores/{wmststoreName}/wmtslayers